source/community/irc/index.html

```diff
@@ -200,27 +227,41 @@ <h3 class="trim-top">Bots</h3>
             some time for it to generate the graph, so be patient.
           </dd>
 
-          <dt><a href="https://github.com/perl6/synopsebot">synopsebot6</a></dt>
+          <dt><a href="https://github.com/Raku/synopsebot">synopsebot6</a></dt>
           <dd>
             Creates links to the synopses and turns mentions of RT ticket
             numbers into clickable RT links.
           </dd>
 
+          <dt><a href="https://github.com/Raku/whateverable/wiki/Tellable">tellable</a></dt>
+          <dd>
+            An IRC bot for passing messages to users who are currently offline. 
+            You can also use it to see when was the last time somebody talked.
+          </dd>
+
+
           <dt><a href="https://github.com/zoffixznet/undercover">Undercover</a></dt>
           <dd>
             Very similar to SourceBaby, except points to
             <a href="https://wtf.rakudo.party/">https://wtf.rakudo.party/</a>
             indicating stresstest coverage of code.
           </dd>
 
-          <dt><a href="https://github.com/perl6/whateverable/wiki/Unicodable">unicodable</a></dt>
+          <dt><a href="https://github.com/Raku/whateverable/wiki/Undersightable">undersightable</a></dt>
```

Comment:
This bot is broken and not currently available.

Reply:
Gotcha. It should be removed now.